app-web/src/components/Home/index.js

```diff
@@ -25,7 +25,8 @@ export const StyledLink = styled(Link)`
   text-decoration: none;
   :hover {
     color: inherit;
-    text-decoration: none;
+    text-decoration: underline;
+    text-decoration-color: #1a5a96;
```

Comment:
is this color not happening by default?

Reply:
it wasn't no, that was for the "Topics" header